wsd/Admin.cpp

```diff
@@ -294,6 +295,10 @@ void AdminSocketHandler::handleMessage(const std::vector<char> &payload)
         // Let's send back the current log levels in return. So the user can be sure of the values.
         sendTextFrame("channel_list " + _admin->getChannelLogLevels());
     }
+    else if (tokens.equals(0, "clusterresize"))
```

Comment:
I'd expect to have something more specific here: "updateroutetoken <URL>" or whatever message - such that we can tell all of our clients about their new URL - without them needing to re-connect to the server.